okapi-core/src/main/java/org/folio/okapi/managers/TenantManager.java

```diff
@@ -673,14 +674,14 @@ private void fireTimer(Tenant tenant, ModuleDescriptor md, RoutingEntry re, Stri
     ModuleInstance inst = new ModuleInstance(md, re, path, httpMethod, true);
     MultiMap headers = MultiMap.caseInsensitiveMultiMap();
     logger.info("timer call start module {} for tenant {}", md.getId(), tenantId);
-    proxyService.callSystemInterface("supertenant", headers, tenant, inst, "", cRes -> {
+    proxyService.callSystemInterface("supertenant", headers, tenant, inst, "", cres -> {
```

Comment:
"cres" is not a correct word, or incorrectly camelCases. You may use "res" instead.